### ISSUE TITLE
ListEventAttendees doesn't always return the current user correctly even when they are attending an event

### ISSUE THREAD

**Issue by darrenvong** [Couchers-org/couchers#1617]
Here are some rough reproduction steps:
1. Create an event
2. Transfer the event to a community (although this step probably not necessarily, only needed for it to show up in UI/`ListEvents` from `communities.proto`
3. Change the current user's attendance state to "not going" (with `SetEventAttendance`)
4. Change the current user's attendance state to "going" again

**Expected behaviour**
`ListEventAttendees` should return the current user's ID

**Actual/current behaviour**
`ListEventAttendees` returns another user's ID. This ID seems to be determined from the row's auto increment ID in `event_occurence_attendees` in the database

Basically can't release the event feature until this is fixed...

Other notes: I suspect `ListEventOrganizers` and `ListEventSubscribers` could also have this same problem, so probably worth double checking that too...
